lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1199,7 +1199,12 @@ def _create_node_runbook(
             image_info = self._get_image_info(
                 azure_node_runbook.location, azure_node_runbook.marketplace
             )
-            azure_node_runbook.hyperv_generation = image_info.hyper_v_generation
+
+            # HyperVGenerationTypes return "V1"/"V2", so we need to strip "V"
+            azure_node_runbook.hyperv_generation = image_info.hyper_v_generation.strip(
```

Comment:
use `int()` to convert it to int, so, it's consistent with the schema.

Reply:
This was already getting handled downstream, so I didn't put it here explicitly